src/components/ScrollableContainer/ScrollableContainer.jsx

```diff
@@ -131,6 +150,11 @@ const shadowShape = PropTypes.shape({
   scrolled: PropTypes.string,
 })
 
+const resizeObserversShape = PropTypes.shape({
```

Comment:
Was there a need to create a variable for this prop?

Reply:
On black friday they were only a nickel a pop so I just said why not

Reply:
😂